app/views/commodities/_commodity.html.erb

```diff
@@ -23,7 +23,10 @@
         <% end %>
         <div class="duty" aria-describedby="commodity-duty-title">
           <% tcd = commodity.overview_measures.third_country_duties %>
-          <% if tcd.count > 0 %>
+
+          <% if tcd.count > 1 %>
+            <%= render '/commodities/additional_code_table', measures: tcd %>
```

Comment:
This should not require a leading slash - I'm not sure about when in a view (and obviously rendering a partial) but in a controller that actually refers to a specific template file rather a relative one